evidence/rossbulat/0002-rossbulat-retain.md

```diff
@@ -0,0 +1,88 @@
+# Evidence-0002: Retention at Rank 1
+
+|                 |                                                                                             |
+| --------------- | ------------------------------------------------------------------------------------------- |
+| **Report Date** | Date of submission (2024/09/21)                                                             |
+| **Submitted by**| Ross Bulat                                                                        |
+
+
+## Member details
+
+- Matrix username: @rossbulat:matrix.org
+- Polkadot address: 1hYiMW8KSfUYChzCQSPGXvMSyKVqmyvMXqohjKr3oU5PCXF
+- Current rank: 1
+- Date of initial induction: 15th July 2024
```

Comment:
The initial induction was on 6th December 2023, as shown here: https://collectives.polkassembly.io/referenda/48